2023_Salazar_MachuPicchu/2023_Salazar_MachuPicchu.janno

```diff
@@ -1,57 +1,57 @@
+SHN-355_MNT	M	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	B2b	Q1b1a1a1i	Tooth (molar)	1	SHN-355	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	763022	n/a	3,50E+13	7.062	1,55E-08	ANGSD[v0.935]	Nr Snps (per library): 2450. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565585;ERR11518578	n/a	n/a	n/a	n/a	SHN-355	SHN-355	2.538.170.805.281	33.548.492.075.770.000	41.562.188.743.068	47.931.167.589.023.400
+SHN-492_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	B2	n/a	Tooth (molar)	1	SHN-492	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	619537	n/a	5,30E+13	168.898	4,17E-07	ANGSD[v0.935]	Nr Snps (per library): 3415. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565586;ERR11518582	n/a	n/a	n/a	n/a	SHN-492	SHN-492	4.321.895.941.449	41.992.200.936.200.000	785.494.195.695.924	5.038.703.697.934.000
+SHN-575_MNT	M	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	A2+(64)	Q1b1a2	Tooth (molar)	1	SHN-575	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	879218	n/a	5,99E+12	6.083	3,38E-09	ANGSD[v0.935]	Nr Snps (per library): 4670. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565587;ERR11518597	n/a	n/a	n/a	n/a	SHN-575	SHN-575	1.977.256.985.031	2.653.676.019.828	409.935.706.178.905	48.719.175.600.644.600
+SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7,06E+12	185.285	6,38E-08	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	6.839.695.315.628	7.178.577.511.500.000	7.962.545.374.461.060	5.962.469.648.282.000
```

Comment:
```suggestion
SHN-902_MNT	F	Cusco City;Cusco;Sacsahuaman	n/a	n/a	n/a	n/a	n/a	n/a	Peru	PE	Cusco City;Cusco	Sacsahuaman	-13,50986	-71,98169	contextual	n/a	n/a	n/a	1400	1466	1533	Dates (CE) taken from Table S1 A	D1h	n/a	Tooth (molar)	1	SHN-902	1240K	half	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.4.6	n/a	327344	n/a	7,06E+12	185.285	6,38E-08	ANGSD[v0.935]	Nr Snps (per library): 916. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB62808;ERS15565588;ERR11518584	n/a	n/a	n/a	n/a	SHN-902	SHN-902	6.839.695.315.628	7.178.577.511.500.000	7.962.545.374.461.060	5.962.469.648.282.000

```

Reply:
Hi, I reverted the changes made to those columns. I am not sure exactly how they happened, as I did not change them intentionally. Might have happened while changing between file formats.